cmd/oci-runtime-tool/generate.go

```diff
@@ -608,6 +613,18 @@ func parseRlimit(rlimit string) (string, uint64, uint64, error) {
 	return parts[0], uint64(hard), uint64(soft), nil
 }
 
+func parseNamespace(ns string) (string, string, error) {
+	parts := strings.SplitN(ns, ":", 2)
```

Comment:
More importantly, as far as I can tell there's no tests of the **validation** -- which is quite worrying.